opencti-platform/opencti-graphql/src/domain/stix.js

```diff
@@ -39,6 +39,9 @@ export const stixDelete = async (context, user, id) => {
   const element = await internalLoadById(context, user, id);
   if (element) {
     if (isStixObject(element.entity_type) || isStixRelationship(element.entity_type)) {
+      // To handle delete synchronization events, we force the forceDelete flag to true, because we don't want delete events to create trash entries on synchronized platforms
+      // THIS IS NOT IDEAL: we ideally would need to add the forceDelete flag to all delete related methods on the API,
```

Comment:
shall we open a new issue to "extend gql API/client-python deletion functions with forceDelete option" and keep track of it for the product team ?

Reply:
that's a good idea yes, I'll create the issue

Reply:
created the issue here #6969